docs/CompatibleSoftware.rst

```diff
@@ -19,164 +19,337 @@ application, please contact ocio-dev as well.
+
+- `Color Nodes <https://learn.foundry.com/nuke/content/reference_guide/color_nodes/color_nodes.html>`__
 
-See the the OCIO user guide `for details on baking ICC profiles for Photoshop
-<userguide-bakelut-photoshop>`__
 
 OpenImageIO
 ***********
 
-`Open Source Image Library / Renderer Texture Engine <http://openimageio.org>`__
+OIIO's C++ and Python bindings include several methods for applying color transforms to whole images, notably functions in the ImageBufAlgo namespace including **colorconvert()**, **ociolook()**, **ociodisplay()**, **ociofiletransform()**. These are also available as part of the *oiiotool* command line utility (--colorconvert, --ociolook, --ociodisplay, --ociofiletransform) and the *maketx* utility for preparing textures also supports **--colorconvert**. From C++, there is additional low-level functionality in the header **OpenImageIO/color.h** that are wrappers for accessing underlying OCIO color configurations and doing color processing on individual pixel values.
 
-Available in the current code trunk. Integration is with makecolortx (allowing
-for color space conversion during mipmap generation), and also through the
-public header `src/include/color.h <http://github.com/OpenImageIO/oiio/blob/master/src/include/color.h>`__ .
+Website : `Open Source Image Library / Renderer Texture Engine <http://openimageio.org>`__
 
-Remaining integration tasks include
-`color conversion at runtime <http://github.com/OpenImageIO/oiio/issues/193>`__ .
 
-C++
-***
+PhotoFlow
+*********
+
+PhotoFlow supports OCIO via a dedicated tool that can load a given configuration and apply the available color transforms. So far the tool has been tested it with the `Filmic <https://github.com/sobotka/filmic-blender>`__ and `ACES <https://opencolorio.org/configurations/aces_1.0.3.html>`__ configs.
+
+Website : `<https://github.com/aferrero2707/PhotoFlow>`__
+
 
-The core OpenColorIO API is available for use in C++. See the `export
-directory
-<http://github.com/AcademySoftwareFoundation/OpenColorIO/tree/master/export/OpenColorIO>`__
-for the C++ API headers.  Minimal code examples are also available in
-the source code distribution. Of particular note are
-`src/apps/ocioconvert/
-<https://github.com/AcademySoftwareFoundation/OpenColorIO/tree/master/src/apps/ocioconvert>`__
-and `src/apps/ociodisplay/
-<https://github.com/AcademySoftwareFoundation/OpenColorIO/tree/master/src/apps/ociodisplay>`__
+Photoshop
+*********
+
+OpenColorIO display luts can be exported as ICC profiles for use in photoshop. The core idea is to create an .icc profile, with a valid description, and then to save it to the proper OS icc directory. (On OSX, ``~/Library/ColorSync/Profiles/``). Upon a Photoshop relaunch, Edit->Assign Profile, and then select your new OCIO lut.
 
-Also see the :ref:`developer-guide`
+Website : `<https://www.adobe.com/products/photoshop.html?promoid=PC1PQQ5T&mv=other>`__
```

Comment:
This link should just be (with promo id part): https://www.adobe.com/products/photoshop.html